Update "Accuracy" article with ground truth information (#6625)
ppy/osu-wiki: wiki/Gameplay/Accuracy/en.md

### ![](/wiki/shared/mode/catch.png) osu!catch

![Accuracy = (number of caught fruits + number of caught drops + number of caught droplets) / (number of all fruits + number of all drops + number of all droplets)](img/accuracy_catch_updated.png "Accuracy formula for osu!catch")

In osu!catch, accuracy is calculated by taking the total number of non-spinner hit objects collected, divided by the total number of non-spinner objects. All hit objects have the same value; except for bananas, as they are part of the spinner objects.

*Notes for API users:*

- The number of caught drops is returned as `count100`.
- The number of caught droplets is returned as `count50`.
- The number of missed fruits *and* drops cumulatively is returned as `countMiss`.
- The number of missed droplets is returned as `countKatu`.

> - `countGeki` should not be used to calculate the accuracy at all. It is the count of caught combo-ending fruits.

Reviewer: Which API? osu! API v1 or v2 or both?

Author: Doesn't matter in this context as things stand. `countGeki` and `countKatu` are present in both versions. However I'm pretty sure api v2 will be eventually migrating to something saner at some point.

Reviewer: Alright add api as hyperlink to this article https://osu.ppy.sh/wiki/en/osu!api

Reviewer 2: ```suggestion
*Notes for [API](/wiki/osu!api) users:*

- The number of caught drops is returned as `count100`.
- The number of caught droplets is returned as `count50`.
- The number of missed fruits *and* drops cumulatively is returned as `countMiss`.
- The number of missed droplets is returned as `countKatu`.
- `countGeki` should not be used to calculate the accuracy at all. It is the count of caught combo-ending fruits.
```

### ![](/wiki/shared/mode/mania.png) osu!mania

> In osu!mania, accuracy is calculated similarly to [osu!](#osu!). However, the weighting of rainbow 300s (also referred to as MAX results) depends on whether Score V2 is active.

Reviewer 2: ```suggestion
In osu!mania, accuracy is calculated similarly to [osu!](#osu!). However, the weighting of rainbow 300s (also referred to as MAX results) depends on whether ScoreV2 is active.
```

> Without score V2 active, rainbow 300s and gold 300s have an equal weight of 300:

Reviewer 2: ```suggestion
Without ScoreV2 active, rainbow 300s and gold 300s have an equal weight of 300:
```

> ![Accuracy = (300 \* (number of MAXs + number of 300s) + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (300 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v1.png "Accuracy formula for osu!mania and score V1")

Reviewer 2: ```suggestion
![Accuracy = (300 \* (number of MAXs + number of 300s) + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (300 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v1.png "Accuracy formula for osu!mania with ScoreV1")
```

> Score V2 increases the weighting of rainbow 300s to 305:

Reviewer 2: ```suggestion
ScoreV2 increases the weighting of rainbow 300s to 305:
```

> ![Accuracy = 305 \* number of MAXs + 300 \* number of 300s + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (305 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v2.png "Accuracy formula for osu!mania and score V2")

Reviewer 2: ```suggestion
![Accuracy = 305 \* number of MAXs + 300 \* number of 300s + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (305 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v2.png "Accuracy formula for osu!mania and ScoreV2")
```

> ![Accuracy = 305 \* number of MAXs + 300 \* number of 300s + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (305 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v2.png "Accuracy formula for osu!mania and ScoreV2")

Reviewer 2: programmer moment

```suggestion
![Accuracy = 305 \* number of MAXs + 300 \* number of 300s + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (305 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v2.png "Accuracy formula for osu!mania with ScoreV2")
```
